app/controllers/jobs_controller.rb

```diff
@@ -7,5 +7,7 @@ def index
 
   def show
     @job = Job.find(params[:id])
+    @job_description =
```

Comment:
When we get real data it will go away but thats the only way to format the seed data properly